web/src/components/tooltips/mapexchangetooltip.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { MAP_EXCHANGE_TOOLTIP_KEY } from '../../helpers/constants';
+import { __ } from '../../helpers/translation';
+import { co2Sub } from '../../helpers/formatting';
+import Tooltip from '../tooltip';
+
+import { CarbonIntensity, ZoneName } from './common';
+
+const mapStateToProps = state => ({
+  colorBlindModeEnabled: state.application.colorBlindModeEnabled,
+  // TODO: Should probably rename tooltipZoneData -> tooltipData
```

Comment:
feel free to do it now

Reply:
Done in https://github.com/tmrowco/electricitymap-contrib/pull/2276/commits/7e8a31a3b316a35dfa7c4745e596305185a79746.

This renaming much less painful than I expected :stuck_out_tongue: